projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/order-history/b2b-order-history-orders-flow.e2e-spec.ts

```diff
@@ -81,4 +82,22 @@ describe('Order History with orders', () => {
     cy.get('.cx-order-history-po a').should('contain', poNumber);
     cy.get('.cx-order-history-cost-center a').should('contain', costCenter);
   });
+
+  it('should display order details page with the reorder button', () => {
+    cy.get('.cx-order-history-value').first().click();
+    cy.get('cx-order-details-reorder div div button').should(
+      'contain',
+      'Reorder'
+    );
+  });
+
+  it('should show result dialog on adding items to the the cart and update the cart', () => {
+    cy.get('cx-order-details-reorder div div button').first().click();
+    cy.get('.cx-cart-mod-entry-container').should('exist');
+
+    cart.goToCart();
+    cart.verifyCartNotEmpty();
```

Comment:
Without adding too much complexity, is there a way to make a more reliable assertion than "the cart is not empty"?